src/hooks/usePaymentInitiator.ts

```
@@ -0,0 +1,808 @@
+            setError('Failed to calculate fees')
+            setFeeCalculations({
+                networkFee: { expected: '0.00', max: '0.00' },
+                slippage: undefined,
+                estimatedFee: '0.00',
+                totalMax: '0.00',
+            })
+        } finally {
+            const timer = setTimeout(() => {
+                setIsCalculatingFees(false)
+            }, 100)
+            return () => clearTimeout(timer)
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)